draft-dekater-scion-controlplane.md

```diff
@@ -943,8 +943,9 @@ The following code block defines the hop entry component `HopEntry` in Protobuf
 ~~~~
 
 - `hop_field`: Contains the authenticated information about the ingress and egress interfaces in the direction of beaconing. The data plane needs this information to forward packets through the current AS. For further specifications, see [](#hopfield).
-- `ingress_mtu`: Specifies the maximum transmission unit (MTU) of the ingress interface of the current AS.
+- `ingress_mtu`: Specifies the maximum transmission unit (MTU) of the ingress interface (in beaconing direction) of the hop being described. Packets traversing the associated inter-AS link in either direction cannot be larger than this size. As a result, no path segment constructed from the containing beacon can have an MTU greater than this. How the control service obtains the MTU of an inter-AS link is implementation dependent. It may be discovered or configured. Current practice to make it a configuration item.
 
+In this description, MTU and packet size are to be understood in the same sense as in {{RFC1122}}. That is, exclusive of any layer 2 framing or packet encapsulation (for links using an underlay network).
```

Comment:
Are you referring to a specific section of RFC1122? I skimmed through it and I did not find where you are referring to immediately. In the terminology section, it only says: 
```
         MTU
              The maximum transmission unit, i.e., the size of the
              largest packet that can be transmitted.
```

Reply:
A precise reference would be complicated. I was shocked to discover how poorly the term is defined. This is the closest I found to a definition and it is spread between the paragraph you quote and the text after it:

> The terms frame, packet, datagram, message, and segment are
>          illustrated by the following schematic diagrams:
> 
>          A. Transmission on connected network:
>            _______________________________________________
>           | LL hdr | IP hdr |         (data)              |
>           |________|________|_____________________________|
> 
>            <---------- Frame ----------------------------->
>                     <----------Packet -------------------->

So, the MTU is defined first as the property of a packet, and then a packet is defined as the portion of the Frame that follows the LL header.

Reply:
Ok, then let's perhaps leave the reference as it is, I think the text is clear enough,